test/drivers/test_suites/syscall_exit_suite/memfd_create_x.cpp

```diff
@@ -0,0 +1,41 @@
+#include "../../event_class/event_class.h"
+
+#include <sys/mman.h>
+
+#ifdef __NR_memfd_create
+
+TEST(SyscallExit, memfd_createX)
+{
+    auto evt_test = get_syscall_event_test(__NR_memfd_create, EXIT_EVENT);
+
+    evt_test->enable_capture();
+
+    /*=============================== TRIGGER SYSCALL ===========================*/
+
+    const char* name = NULL;
```

Comment:
probably we need a test for the failure case and one for the success one... see `openX_success` and `openX_failure`